keymaps/intellij-idea-keymap.cson

```diff
@@ -249,6 +249,13 @@
   # Move line down
   'ctrl-shift-down': 'editor:move-line-down'
 
+  # Delete to previous word
+  'ctrl-backspace': 'editor:delete-to-previous-word-boundary'
```

Comment:
I just checked the https://resources.jetbrains.com/assets/products/intellij-idea/IntelliJIDEA_ReferenceCard_mac.pdf reference card and the `ctrl-backspace` from your windows shortcut is `alt-backspace` on the mac. And this is already supported on the Mac so you do not need to add this there.

And could you maybe rename the comment from `Delete to previous word` to `Delete to word start` as this is the official name in the reference card. This makes it easier to search for the shortcuts.

And for the Atom shortcut I think the `editor:delete-to-beginning-of-word` fits better as I cannot find your `editor:delete-to-previous-word-boundary` in my default Atom installation.